src/Element/Element.php

```diff
@@ -204,6 +196,6 @@ public function getOuterHtml()
      */
     protected function elementNotFound($type, $selector = null, $locator = null)
```

Comment:
I'm wondering whether we should throw exceptions in place now and deprecate this method (this was added in 1.6 just so that our own child classes can avoid using the deprecated Session getter)